libsolidity/interface/CompilerStack.h

```diff
@@ -289,7 +290,8 @@ class CompilerStack: boost::noncopyable
 	std::vector<Source const*> m_sourceOrder;
 	std::map<std::string const, Contract> m_contracts;
 	std::string m_formalTranslation;
-	ErrorList m_errors;
+	ErrorList m_errorList;
```

Comment:
I think it's fine to roll in here.  I'm already changing the constructor by adding more initializers.